src/components/InputV2/Filecoin.tsx

```diff
@@ -0,0 +1,159 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import { FilecoinNumber } from '@glif/filecoin-number'
+import { BaseInput, BaseInputProps } from './Base'
+import { FILECOIN_NUMBER_PROP } from '../../customPropTypes'
+
+type FilecoinDenomination = 'fil' | 'picofil' | 'attofil'
```

Comment:
Would be nice to pull this from the `FilecoinNumber` package itself, but it's not exported... grrr https://github.com/glifio/modules/blob/primary/packages/filecoin-number/src/FilecoinNumber.ts#L7